xee/ext_integration_test.py

```diff
@@ -41,17 +41,14 @@
 ]
 
 
-def _read_identity_pool_creds() -> identity_pool.Credentials:
```

Comment:
Happy to revert to this, but it was easier for me to get tests up and running with  `google.auth.default`, which looks for `GOOGLE_APPLICATION_CREDENTIALS` by default anyways.